src/spikeinterface/benchmark/benchmark_plot_tools.py

```diff
@@ -1,47 +1,148 @@
-import numpy as np
 import warnings
 
+import numpy as np
+from spikeinterface.widgets import get_some_colors
+from .benchmark_tools import sigmoid, fit_sigmoid
+
+
+def despine(ax_or_axes):
```

Comment:
not sure to be happy to have more seaborn dependency.

Reply:
we have it in other plots as well, like swarm plots. It's optional